includes/Core/Modules/Tags/Module_Web_Tag.php

```diff
@@ -61,6 +61,31 @@ public function get_tag_blocked_on_consent_attribute() {
 		return '';
 	}
 
+	/**
+	 * Gets the array of HTML attributes for a script tag that may potentially require user consent before loading.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array containing HTML attributes to add if the tag requires consent to load, or an empty array.
+	 */
+	public function get_tag_blocked_on_consent_attribute_array() {
```

Comment:
@aaemnnosttv I agree that it would make sense to incorporate this into `Blockable_Tag_Interface`. The question is under what name? In a way the current `get_tag_blocked_on_consent_attribute` is already confusing since it doesn't return an attribute but a string of multiple attributes.

While this would be a technically breaking change, I'd recommend renaming the existing interface method to `get_tag_blocked_on_consent_attributes_string` (since that is the more specific return value) and introduce the method here as `get_tag_blocked_on_consent_attributes`.

Reply:
`Blockable_Tag_Interface` is also used by other modules that does not yet use script tags and therefore doesn't fall under this issue. Should this refactor be included in this PR or a separate PR? @felixarntz @aaemnnosttv

Reply:
@felixarntz the thing is we don't need both methods to be part of the interface (and thus required to be implemented by the consumers). We can either extract attributes from the string returned by the method or build the string from the attributes returned by a different method, but we shouldn't require both in the same interface I don't think.

@kuasha420 let's handle the interface changes in a separate issue to not block this one where we can think more about if/how we want to change that interface.